yunwoo/programmers-1/옹알이-(2).js

```diff
@@ -0,0 +1,21 @@
+function solution(babbling) {
+  let result = 0;
+
+  babbling.forEach((el) => {
+    const reg = /(aya|ye|woo|ma)/g;
+    const repeatReg = /(ayaaya|yeye|woowoo|mama)/g;
+    const DeleteRepeatbabbling = el.replace(repeatReg, "No");
+    const getbabbling = DeleteRepeatbabbling.replace(reg, "");
+
+    if (!getbabbling) result++;
+  });
+
+  return result;
+}
```

Comment:
```suggestion
function solution(babbling) {
  const reg = /(aya|ye|woo|ma)/g;
  const repeatReg = /(ayaaya|yeye|woowoo|mama)/g;
  let result = 0;

  babbling.forEach((el) => {
    const DeleteRepeatbabbling = el.replace(repeatReg, "No");
    const getbabbling = DeleteRepeatbabbling.replace(reg, "");

    if (!getbabbling) result++;
  });

  return result;
}
```
- 훌륭한 풀이 잘봤습니다 🙆‍♂️ 단 정규표현식은 forEach 외부에서 선언해주셔도 됩니다! 매번 forEach로 반복할 때마다 선언하는 것은 불필요한 작업이겠죠?
- 이전에 옹알이1할 때 제가 추후에 공유드렸던 내용입니다 🙇‍♂️